tutorials/governance/opengov/create-a-referenda.md

```diff
@@ -0,0 +1,91 @@
+
+The example below demonstrates the creation of a preimage to propose and approve the spending of treasury funds.
+
+To submit a preimage, follow these steps:
+
+1. Visit [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws#/explorer), navigate to the **Governance** dropdown and select the **Preimages** option
+   
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-1.webp)
+
+2. Click in the **+ Add preimage** button
+   
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-2.webp)
+
+3. Construct the extrinsic you want to execute and submit the preimage:
+    1. Select the **treasury** pallet
+    2. Choose the **spendLocal** extrinsic
+    3. Fill in the parameters:
+        - **amount** - amount to be transferred from the treasury to the `beneficiary`
+        - **beneficiary** - destination account for the transfer
+    4. Copy the preimage hash -
+    the preimage hash for this example is `0x0e634ee7f6adf6e7cfb7e2fa6c77454c3d0ce46f64287df0c1da8d079d9633a2`
+    5. Click the **+ Submit preimage** button
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-3.webp)
+
+## Submit a Proposal
+
+Submitting a proposal requires you to bond a specified amount of tokens. The bond value depends on the configuration settings of the chain you use.
+
+Before submitting the proposal, you must first provide the preimage hash. The preimage hash is a cryptographic hash of the full proposal details you intend to enact.
+
+To submit a proposal, follow these steps:
+
+1. Navigate to the **Governance** dropdown and select **Referenda**
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-4.webp)
+2. Click in the **+ Submit proposal** button
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-5.webp)
+3. Fill in the parameters and submit it:
+    1. Select the account from which the proposal will be registered. The balance lock will be applied to this account
+    2. Choose your proposal's origin (and, by extension, the track). Each track has different durations, privileges, and acceptance criteria
+    3. Enter the preimage hash obtained in previous steps. The preimage length field will be automatically populated
+    4. Select the enactment delay. This can be specified either as a block number or as a specific number of blocks after the referendum is approved
+    5. Click the **Submit proposal** button
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-6.webp)
+4. After submitting, you should see a `referenda.Submitted` event with your proposal in the explorer section. The proposal number will be displayed in the **index** field
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-7.webp)
```

Comment:
Again, super nitpicking here. Could you please take a look at how step 'A' overlaps with the circle of the accounts input? It seems a bit off to me. Maybe you could consider moving the step 10px to the top, as well as all the other elements, to ensure consistency in the UI